index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Graffiti bootcamp - project 3</title>
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="style.css">
+  <!-- Google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Raleway:300,400" rel="stylesheet">
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+</head>
+
+<body>
+  <header>
+    <!-- video or image as a header is cool :) -->
+    <div id=hero>
+      <video autoplay loop muted preload="auto" id="graffitiVideo">
+        <source src="./videos/compressed_graffiti.mp4" type="video/mp4">
+        Your browser doesn't support HTML5 video.
+      </video>
+      <div class="content">
+        <h1>Graffiti artist bootcamp</h1>
+        <h2> Want to become a real graffiti artist ? Sign up to the graffit artist bootcamp this fall 2021.</h2>
+        <p><button id=signbutton class="w3-button w3-black w3-round-xxlarge">Sign up to Graffiti bootcamp</button></p>
+      </div>
```

Comment:
`content` is a very general class name.  If you saw 
```css
content {
```
In your CSS file, without any context it may be difficult to understand.  You might gain clarity if you call it `header-content`